tools/reach-calculator.html

```diff
@@ -86,21 +154,168 @@ <h2>Site Breakage Reach Calculator</h2>
+                <dt>Additional unaccounted-for breakage:</dt><dd>High confidence</dd>
+                <dt class="answer">Final % of page loads:</dt><dd>1.68%</dd>
+            </dl>
+
+            <h3>Single API example: WebShare API not supported</h3>		
+            <dl>
+                <dt>API usage share:</dt><dd>0.07%</dd>
+                <dt class="answer">Initial % of all page loads:</dt><dd><a href="https://chromestatus.com/metrics/feature/timeline/popularity/2737" target="_blank">0.07%</a></dd>
+                <dt>Further reduction assumption:</dt><dd>[None]</dd>
+                <dt class="answer">Updated % of page loads:</dt><dd>0.07%</dd>
+                <dt>Additional unaccounted-for breakage:</dt><dd>Low confidence</dd>
+                <dt class="answer">Final % of page loads:</dt><dd>0.07%</dd>
+            </dl>
+
+            <h3>Extreme example 1: Google is broken for everyone</h3>		
+            <dl>
+                <dt>Domain rank:</dt><dd>1</dd>
+                <dt class="answer">Initial % of all page loads:</dt><dd>21.352%</dd>
+                <dt>Further reduction assumption:</dt><dd>[None]</dd>
+                <dt class="answer">Updated % of page loads:</dt><dd>21.352%</dd>
+                <dt>Additional unaccounted-for breakage:</dt><dd>Low confidence</dd>
+                <dt class="answer">Final % of page loads:</dt><dd>21.352%</dd>
+            </dl>
+
+            <h3>Extreme example 2: Videos not playing on my personal blog	</h3>		
+            <dl>
+                <dt>Domain rank:</dt><dd>1,000,000</dd>
+                <dt class="answer">Initial % of all page loads:</dt><dd>0.00011%</dd>
+                <dt>Further reduction assumption:</dt><dd>10%</dd>
+                <dt class="answer">Updated % of page loads:</dt><dd>0.000011%</dd>
+                <dt>Additional unaccounted-for breakage:</dt><dd>High confidence</dd>
+                <dt class="answer">Final % of page loads:</dt><dd>0.0011%</dd>
+            </dl>
+        </p>
+    </details>
+    
+    <details>
+        <summary>How is the page load percentage calculated from domain rank?</summary>
+        <p>The paper "<a href="https://dl.acm.org/doi/10.1145/3517745.3561418" target="_blank">A World Wide View of Browsing the World Wide Web</a>", written by researchers at Google and Stanford University, presents data from "several hundred million global users" revealing the following traffic statistics:</p>
+        <ul>
+            <li>The top website has 17% of traffic</li>
+            <li>The top 6 sites have 25% of traffic</li>
+            <li>The top 100 sites have 40% of traffic</li>
+            <li>The top 10K sites have 70% of traffic</li>
+            <li>The top 1M sites have 95%+ of traffic</li>
+        </ul>
+        <p>This is a logarithmic increase which, when plotted, gives us the following trend line formula:</p>
+
+        <math xmlns="http://www.w3.org/1998/Math/MathML">
```

Comment:
Hah this is massively above and beyond here! I just meant copying that comment into a <!--- tag. Anyway all good!